pkg/ebpf/c/tcp_states.h

```diff
@@ -0,0 +1,23 @@
+/* Source: include/net/tcp_states.h */
+
+#ifndef _LINUX_TCP_STATES_H
+#define _LINUX_TCP_STATES_H
+
+enum {
+	TCP_ESTABLISHED = 1,
+	TCP_SYN_SENT,
+	TCP_SYN_RECV,
+	TCP_FIN_WAIT1,
+	TCP_FIN_WAIT2,
+	TCP_TIME_WAIT,
+	TCP_CLOSE,
+	TCP_CLOSE_WAIT,
+	TCP_LAST_ACK,
+	TCP_LISTEN,
+	TCP_CLOSING,
+	TCP_NEW_SYN_RECV,
+
+	TCP_MAX_STATES
+};
```

Comment:
Can we not pull these in directly from the kernel headers ?

Reply:
Err yes 🤦

Reply:
Discussed offline and realized:
- this would make it so the system-probe go build could reference kernel headers where it couldn't before (which is not good)
- an effect of that would be that all the CI images would need to pull in kernel headers
 

This enum is going to remain the same (it's a part of the [/proc api](https://stackoverflow.com/questions/5992211/list-of-possible-internal-socket-statuses-from-proc)) so we can leave this as is.